_specs/invoke-signer.md

```diff
@@ -0,0 +1,113 @@
+## Standardize OS Level Management of Signature Handling
+
+    ECLIP: <pending>
```

Comment:
```suggestion
    ECIP: 1067
```

Reply:
Thanks, just going to wait 24 hours to see if it gets an EIP number instead. If not, I will commit this one